README.md

```diff
@@ -21,19 +21,20 @@ Mocca is a GraphQL client for JVM languages with the goal of being easy to use,
 Mocca offers support for:
 
 1. GraphQL features
-	1. GraphQL queries and mutations
-	1. Automatic variable definition
-	1. Automatic selection set definition based on DTO response type
-	1. Annotation and String based custom input variables
-	1. Annotation and String based custom selection set
-1. Static and dynamic HTTP request headers
-1. Observability via Micrometer
-1. Resilience with via Resilience4J
-1. Flexible API allowing various pluggable HTTP clients
-1. Asynchronous support
-	1. CompletableFuture
-	1. Pluggable asynchronous HTTP clients
-	1. User provided executor services
+    1. GraphQL queries and mutations
+    1. Automatic variable definition
+    1. Automatic selection set definition based on DTO response type
+    1. Annotation and String based custom input variables
+    1. Annotation and String based custom selection set
+2. Static and dynamic HTTP request headers
```

Comment:
We had left numbering here intentionally as 1 in all items. When we do that, we leave it to the render to define the actual line number, which makes maintaining the md file easier, besides avoiding wrong numbers. Having said that, could you revert it to back to how it was?

Reply:
Sorry about that. the IJ markdown plugin insisted on renumbering. Maybe you know how to get around that. I can go in and re do that with vi or something not quite so *INTELLIJENT*

Reply:
haha